### DISCUSSION TITLE
What's the best way to implement Google place autocomplete for React Native Web?

### DISCUSSION THREAD

**Discussion by thecannabisapp** [react-hook-form/react-hook-form#8159] [Q&A]
Hi,

I'm trying to build a signup form with an address field that I'd like to autocomplete using the google place autocomplete API.

The form is being built using Expo 44, RHF, and React Native Paper TextInput components. I've searched around and didn't find anything that matches the set-up I'm using.

Just wondering if anyone has previously implemented this feature using the same setup and if you have any tips to share.

Thank you,

Jia

**Comment by Moshyfawn** [comment]
Do you mind creating a codesanbox with the basic autocomplete API and RN Web setup?

Here's [a codesanbox template](https://codesandbox.io/s/interesting-wilbur-xp6vn7) for starters
